pkg/agent/cniserver/ipam/ipam_delegator_test.go

```diff
@@ -0,0 +1,185 @@
+//go:build linux
```

Comment:
why make specific to linux?

Reply:
It will try to locate the CNI path in `/opt/cni/bin`, so when running on Windows, it will get an error, so I specific the test to linux.

Reply:
Why is the real IPAM plugin required? Can you use mock functions?

Reply:
OK, I'll try to use mock functions to avoid this.

Reply:
@wenyingd @tnqn I mocked the `defaultExec` and then we can remove the `go:build linux` now. In order to mock it, I have to change the `defaultExec` type from struct `invoke.DefaultExec` to interface `invoke.Exec`